src/clients/api/mutations/exitMarket.ts

```diff
@@ -0,0 +1,19 @@
+import { Contract } from 'web3-eth-contract';
+
+export interface IExitMarketInput {
+  comptrollerContract: Contract; // @TODO: use contract type (through Typechain?)
+  account: string | undefined | null;
+  vtokenAddress: string;
+}
+
+export type ExitMarketOutput = void;
```

Comment:
No we should not. I am not clear on the benefit of putting the I in front of interfaces. It is new to me and I can look into it. This was something that Dmitriy and you seemed to be interested in. There is even an eslint plugin to enforce it. 

These types are defined correctly. We can change the naming convention to drop the I.

Reply:
This is what eslint says `It can be hard to differentiate between classes and interfaces. Prefixing interfaces with "I" can help telling them apart at a glance.``

This is motivated by the ability to use classes as types.